token-metadata/program/src/state/collection.rs

```diff
@@ -55,6 +55,10 @@ impl CollectionAuthorityRecord {
 #[cfg_attr(feature = "serde-feature", derive(Serialize, Deserialize))]
 #[derive(BorshSerialize, BorshDeserialize, PartialEq, Eq, Debug, Clone)]
 pub enum CollectionDetails {
+    #[deprecated(
+        since = "1.13.0",
```

Comment:
I think we need to update the version to the actual version where the deprecated message will show up.